lms/templates/admin/organization/usage.html.jinja2

```diff
@@ -7,11 +7,6 @@
         {{ macros.organization_preview(request,org) }}
     </fieldset>
 
-    <fieldset class="box mt-6">
-      <legend class="label has-text-centered">Company</legend>
-      {{ macros.organization_preview(request, company) }}
```

Comment:
Note that this should have been `company_preview`. Instead of fixing it just removing it and we'll be showing this info in the main page.

I reckon showing in "usage" was the wrong decision.